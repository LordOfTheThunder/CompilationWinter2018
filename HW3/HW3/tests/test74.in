int ManyReturns() {
	if (true) {
		if (true) {
			if (true) {
				return 10;
			}
			return 35b;
		}
		return true;
	}
}

void main() { print("I hate this course"); }

