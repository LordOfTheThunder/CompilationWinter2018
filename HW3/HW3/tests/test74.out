---end scope---
---end scope---
---end scope---
---end scope---
line 9: type mismatch
